HACKING.md

```diff
@@ -26,7 +26,7 @@ Once a release has been created, it can be pushed:
 To cut an official tag release, we generally use the images built by [ci.openshift.redhat.com](https://ci.openshift.redhat.com)
 under the openshift3_ami job.
 
-1. Create a new git tag `git tag v0.X.X -a -m "v0.X.X" HEAD`
```

Comment:
I don't think we would run this for the process described here.  I don't see us doing this as part of origin releases.